src/core_plugins/kbn_vislib_vis_types/public/controls/gauge_options.html

```diff
@@ -35,6 +35,17 @@
     </div>
   </div>
 
+  <div class="kuiSideBarFormRow">
+    <label class="kuiSideBarFormRow__label" for="displayWarnings">
+      Display warnings &nbsp;
+      <kbn-info info="Turns on/off warnings. When turned on, warning will be shown for each term that cannot be matched to a shape in the vector layer based on the join field. When turned off, these warnings will be turned off."></kbn-info>
```

Comment:
"**a** warning will be shown" or "warning**s** will be shown"

Reply:
This is not the correct warning message it seems like. Something like

Consider:
> Turns on/off warnings. When turned on, a warning will be shown if not all labels could be displayed.